lib/ssl/test/ssl_api_SUITE.erl

```diff
@@ -597,6 +600,49 @@ select_sha1_cert(Config) when is_list(Config) ->
                                                    {key, {namedCurve, secp256r1}}]}}),
     test_sha1_cert_conf(Version, TestConfRSA, TestConfECDSA, Config).
 
+%%--------------------------------------------------------------------
+root_any_sign() ->
+    [{doc,"Use cert signed with unsported signature for the root will suceed, as it is not verified"}].
+
+root_any_sign(Config) when is_list(Config) ->
+    Version = ssl_test_lib:protocol_version(Config),
+    #{client_config := CSucess, server_config := SSucess} =
```

Comment:
maybe write a code comment in line 609 and 624 to explain what is relevant in those configs from test perspective;
please explain why 2nd one is expected to fail

Reply:
fixed